### DISCUSSION TITLE
How to enable legacy logout endpoint in legacy KC 18 using wildfly cli ?

### DISCUSSION THREAD

**Discussion by fidgi** [keycloak/keycloak#13153] [Q&A]
Hi,

Since KC 18, the legacy logout endpoint is disable by default (as stated here https://www.keycloak.org/docs/latest/upgrading/index.html#openid-connect-logout).

I'm looking for a way to enable the legacy logout endpoint in KC 18.0.2-legacy using a Wildfly cli .

Any clue will be appreciated.

Regards,

**Comment by bs-matil** [comment]
tbh. I don't know what you are referring to. The logout is enabled as before. The behaviour changed.

**Reply by fidgi** [reply]
Hi ,

Starting from KC18 this is the OpenID Connect compliant endpoint that is enable by default and not the "legacy" endpoint with the "redirect_uri" parameter. 
What I'd like is to enable the "legacy" logout endpoint within the "legacy" distribution of KC18 and via a Jboss cli

**Comment by fidgi** [comment]
Hi, 
I guess the answer is here : https://keycloak.discourse.group/t/keycloak-18-legacy-backward-logout-compatibility-option/15918
